packages/MonorepoBuilder/packages/Split/src/Command/SplitCommand.php

```diff
@@ -59,6 +60,13 @@ protected function configure(): void
                 'monorepo-builder.yml'
             )
         );
+        $this->addOption(
+            'max-processes',
+            null,
+            InputOption::VALUE_REQUIRED,
+            'Maximum number of processes to run in parallel',
+            0
```

Comment:
Why `0`? That will slow whole split process a lot by default.

I'd go to default `8` to keep it fast

Reply:
I chose `0` as the default value for "no limit" so the current behaviour will be unchanged. When `0` is passed for the maximum number, the condition inside of the loop where the processes are created won't ever be fullfilled and therefore it just creates all required processes without waiting for any, just like before.
Maybe a hint like `Maximum number of processes to run in parallel (0 = no limit)` would help?

Reply:
I removed the default value: 266f99baf